go/externals/proof_support_generic.go

```diff
@@ -0,0 +1,97 @@
+package externals
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	libkb "github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+	jsonw "github.com/keybase/go-jsonw"
+)
+
+//=============================================================================
+// GenericProof
+//
+
+type GenericChecker struct {
+	proof libkb.RemoteProofChainLink
+}
+
+var _ libkb.ProofChecker = (*GenericChecker)(nil)
+
+func NewGenericChecker(p libkb.RemoteProofChainLink) (*GenericChecker, libkb.ProofError) {
+	return &GenericChecker{p}, nil
+}
+
+func (rc *GenericChecker) GetTorError() libkb.ProofError { return nil }
+
+func (rc *GenericChecker) CheckStatus(m libkb.MetaContext, h libkb.SigHint, _ libkb.ProofCheckerMode, pvlU libkb.PvlUnparsed) libkb.ProofError {
+	// TODO will have to use the `check_url`/`check_path` config values to verify the proof.
+	return nil
```

Comment:
Can this also `return fmt.Errorf("Not implemented")`? If that breaks then this is fine for now.